app/src/main/java/org/schabi/newpipe/local/feed/FeedViewModel.kt

```diff
@@ -28,6 +30,10 @@ class FeedViewModel(
     initialShowPlayedItems: Boolean = true
 ) : ViewModel() {
     private var feedDatabaseManager: FeedDatabaseManager = FeedDatabaseManager(applicationContext)
+    private var sharedPreferences = PreferenceManager.getDefaultSharedPreferences(applicationContext)
+    companion object {
+        const val SHOW_PLAYED_ITEMS_PREFERENCE = "show_played_items_preference_tag"
```

Comment:
This should be moved into ``settings_keys.xml``

Reply:
hey @litetex I made the change to move the key to settings_keys.xml . let me know if anything else is required